packages/orchestration/src/util/filtering.test.ts

```diff
@@ -0,0 +1,214 @@
+import {
+  buildAzureContentFilter,
+  buildAzureContentSafetyFilter
+} from './filtering.js';
+import { constructCompletionPostRequest } from './request-config.js';
+import type { OrchestrationModuleConfig } from '../orchestration-types.js';
+import type {
+  CompletionPostRequest,
+  FilterConfig,
+  FilteringModuleConfig
+} from '../client/api/schema/index.js';
+
+describe('Content filter util', () => {
+  // TOOD: Remove this test collection once `buildAzureContentFilter` is removed.
+  describe('buildAzureContentFilter', () => {
```

Comment:
I believe we dont need to test this anymore as it's deprecated. It's not like we're going to provide a bug fix/update here ? @tomfrenken

Reply:
I would vote to remove the tests once the function gets removed.

Reply:
The thing is it will only be removed in the next major version and I dont want unnecessary tested code building up in the codebase